authz/rbac_translator.go

```diff
@@ -266,6 +281,37 @@ func parseRules(rules []rule, prefixName string) (map[string]*v3rbacpb.Policy, e
 	return policies, nil
 }
 
+func parseAuditLoggingOptions(options auditLoggingOptions) (*v3rbacpb.RBAC_AuditLoggingOptions, error) {
+	optionsRbac := v3rbacpb.RBAC_AuditLoggingOptions{}
+
+	if options.AuditCondition == "" {
```

Comment:
nit: The enum always defaults to none. You could do
```
if ... != "" {
  rbacCondition, ok := ...
}

Reply:
Doesn't that just flip the if/else?